.circleci/config.yml

```
@@ -98,9 +132,11 @@ jobs:
             echo "source /tmp/venv/openfisca_france/bin/activate" >> $BASH_ENV
 
       - run:
-          name: Test the Web API
+          name: Lint files
           command: |
-            .circleci/test-api.sh
+            make check-syntax-errors check-style
+            .circleci/lint-changed-python-files.sh
+            .circleci/lint-changed-yaml-tests.sh
```

Maybe we could improve logs readability with something like this? 🙂 

```suggestion
     - run: make check-syntax-errors
     - run: make check-style
     - run: .circleci/lint-changed-python-files.sh
     - run: .circleci/lint-changed-yaml-tests.sh
```